scripts/tests/chiptest/__init__.py

```diff
@@ -18,13 +18,26 @@
 import logging
 import os
 import subprocess
+import sys
 from dataclasses import dataclass
 from pathlib import Path
 from typing import Iterator, Set
 
-from . import linux, runner
+from . import runner
 from .test_definition import ApplicationPaths, TestDefinition, TestTag, TestTarget
 
+__all__ = [
+    "TestTarget",
+    "TestDefinition",
```

Comment:
What happened to AllTests?

Reply:
I've removed it since there is no such class in that file (error reported by pylance).